bluesky/tests/test_vertical_integration.py

```diff
@@ -1,17 +1,33 @@
-
-from metadatastore.utils.testing import mds_setup, mds_teardown
-from dataportal import DataBroker as db
+from nose import SkipTest
 from bluesky.examples import stepscan, det, motor
-from bluesky.standard_config import gs
 
 def setup():
-    mds_setup()
+    try:
+        from metadatastore.utils.testing import mds_setup
+    except ImportError:
+        pass  # test will be skipped
```

Comment:
I guess it is not incorrect if enough context is placed around the in-line comment. I initially assumed that this was just a copy/paste error from the `test_scan_and_get_data` function